### ISSUE TITLE
Dask-labextension appears to be incompatible with Jupyter Lab v2.0.0

### ISSUE THREAD

**Issue by andersy005** [dask/dask-labextension#118]
I updated Jupyter Lab to `v2.0.0`, and when I tried updating the labextension, I got a message that there was no compatible version available. As a result, my existing installation of the extension seems to be broken (it doesn't work, which is probably expected). I am wondering whether this is a known issue or I am missing something?

<img width="426" alt="Screen Shot 2020-03-01 at 12 48 55 PM" src="https://user-images.githubusercontent.com/13301940/75632618-25d5c500-5bbb-11ea-951a-b66d1c859840.png">

```bash
abanihi at casper26 in ~
$ jupyter labextension list
JupyterLab v2.0.0
Known labextensions:
   app dir: /glade/work/abanihi/softwares/miniconda3/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0-rc.3  enabled  OK
        @jupyterlab/toc v1.0.1  enabled   X
        @pyviz/jupyterlab_pyviz v0.8.0  enabled   X
        dask-labextension v1.0.1  enabled   X
        jupyterlab-nvdashboard v0.1.9  enabled   X

   The following extension are outdated:
        @jupyterlab/toc
        @pyviz/jupyterlab_pyviz
        dask-labextension
        jupyterlab-nvdashboard

   Consider running "jupyter labextension update --all" to check for updates.


abanihi at casper26 in ~
$ jupyter labextension update --all
Extension '@jupyter-widgets/jupyterlab-manager' already up to date
No compatible version found for jupyterlab-nvdashboard!
No compatible version found for @jupyterlab/toc!
No compatible version found for @pyviz/jupyterlab_pyviz!
No compatible version found for dask-labextension!
```

**Comment by jsignell** [comment]
Yeah it doesn't seem terribly surprising that we'd have to do something to catch up. It seems like it might just need some updated dependencies. I'll take a look tomorrow unless someone else gets a chance before that.

**Comment by ian-r-rose** [comment]
PR open at #119 in case somebody wants to take a look.

**Comment by ian-r-rose** [comment]
Fixed by #119
